### ISSUE TITLE
Move astroML.plotting.setup_text_plots to astroML_figures

### ISSUE THREAD

**Issue by bsipocz** [astroML/astroML#138]
The plots have been moved already, keep the MPL settings with them.

**Comment by brunofisico** [comment]
this functionality, using Latex, was actually removed?

**Comment by bsipocz** [comment]
No, it can be opted out to use Latex, but the figures are still generated with it. The issue is about moving the plot settings to the other repo where all the figures got moved.
